catalogue_pipeline/matcher/src/test/scala/uk/ac/wellcome/platform/matcher/fixtures/MatcherFixtures.scala

```diff
@@ -70,6 +75,13 @@ trait MatcherFixtures
     }
   }
 
+  implicit val instantLongFormat: AnyRef with DynamoFormat[Instant] =
```

Comment:
You should be able to get rid of this block by importing `uk.ac.wellcome.storage.dynamo._` I think